app/src/main/java/com/fstyle/structure_android/screen/main/MainPresenter.java

```diff
@@ -47,23 +48,22 @@ public void searchUsers(int limit, String keyWord) {
         if (!validateDataInput(limit, keyWord)) {
             return;
         }
-        Subscription subscription = mUserRepository.getRemoteDataSource()
-                .searchUsers(limit, keyWord)
+        Subscription subscription = mUserRepository.searchUsers(limit, keyWord)
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
-                .subscribe(users -> mMainViewModel.searchUsersSuccess(users),
-                        throwable -> mMainViewModel.searchError(throwable));
+                .subscribe(users -> mMainViewModel.onSearchUsersSuccess(users),
+                        throwable -> mMainViewModel.onSearchError(throwable));
```

Comment:
ERROR: 'lambda arguments' have incorrect indentation level 24, expected level should be 20.